README.md

```diff
@@ -100,39 +86,39 @@ select where to load the config files from. Typesafe Config has [well-documented
 loading](https://github.com/typesafehub/config#standard-behavior) which we'll not repeat. Please see Typesafe
 Config's documentation for a full telling of the subtleties.
 
-Alternatively, PureConfig also provides a `loadConfigFromFiles` method, which builds a configuration from
+Alternatively, PureConfig also provides a `loadConfigFromFiles` method that builds a configuration from
 an explicit list of files. Files earlier in the list have greater precedence than later ones. Each file can
 include a partial configuration as long as the whole list produces a complete configuration. For an example,
 see the test of `loadConfigFromFiles` in
 [`ApiSuite.scala`](https://github.com/pureconfig/pureconfig/blob/master/core/src/test/scala/pureconfig/ApiSuite.scala).
 
-Because PureConfig uses Typesafe Config to load configuration, it supports reading files in [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md#hocon-human-optimized-config-object-notation), JSON, and Java `.properties` formats. HOCON is a delightful superset of both JSON and `.properties` that is highly recommended. As an added bonus it supports [advanced features](https://github.com/typesafehub/config/blob/master/README.md#features-of-hocon) like variable substitution and file sourcing.
+Because PureConfig uses Typesafe Config to load configurations, it supports reading files in [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md#hocon-human-optimized-config-object-notation), JSON, and Java `.properties` formats. HOCON is a superset of both JSON and `.properties` that is highly recommended. As an added bonus it supports [advanced features](https://github.com/typesafehub/config/blob/master/README.md#features-of-hocon) like variable substitution and file sourcing.
 
 
 ## Documentation
 
-Read the [documentation](https://pureconfig.github.io/docs) website.
+The full documentation can be found [here](https://pureconfig.github.io/docs).
```

Comment:
In terms of SEO it used to be much more effective to have the text of a link describe the target page, rather than be a word like "here." I don't know if that's still the case. 
If my old assumption is still true I'd say something like:
> Please see the [full PureConfig documentation](https://pureconfig.github.io/docs) for more information.